runtime-light/stdlib/instance-cache/instance-cache-state.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "common/mixin/not_copyable.h"
+
+#include "runtime-light/stdlib/instance-cache/request-cache.h"
+
+struct InstanceCacheState : private vk::not_copyable {
+  instance_cache::RequestCache request_cache;
+  static InstanceCacheState& get() noexcept;
+};
```

Comment:
Hmmm... It seems worth to include `InsertNewlineAtEOF` in clang-format

Reply:
Fixed

Reply:
I propose to enable for the full project. It might hit a couple files, but it's not a big deal.